engines/sci/graphics/font.cpp

```diff
@@ -38,7 +38,9 @@ GfxFontFromResource::GfxFontFromResource(ResourceManager *resMan, GfxScreen *scr
 
 	_resource = resMan->findResource(ResourceId(kResourceTypeFont, resourceId), true);
 	if (!_resource) {
-		error("font resource %d not found", resourceId);
+		// The Lighthouse demo does not contain any fonts
```

Comment:
Update: Here's the Lighthouse demo I'm referring to:

http://www.scummvm.org/frs/demos/sci/lighthouse-win-ni-demo-en.zip